README.md

```diff
@@ -74,3 +74,15 @@ run this code in external terminal to run tests
  poetry run
 pytest path/to/test_file
 ```
+## Docker
+To build both production and development run
+```
+$ docker build --target development --tag todo-app:dev .
+$ docker build --target production --tag todo-app:prod .
+```
+and these commands to run it
+```
+docker run --env-file ./.env -p 5100:80 --mount type=bind,source="$(pwd)"\todo_app,target=/app/todo_app todo-app:dev
```

Comment:
This command doesn't quite work:
* it suggests your app is running on port 80, but it's actually running on 5000
* the bind mount doesn't line up with your actual directories inside the container
  * your code is at `/todo_app`, so for example your app.py file is at `/todo_app/todo_app/app.py` 
  * the target for this bind mount is at `/app/todo_app`
  * so you're adding a new app.py file at `/app/todo_app/app.py` instead of replacing the one that's being used